be/src/exprs/bloomfilter_predicate.h

```diff
@@ -334,6 +334,12 @@ struct DateTimeFindOp : public CommonFindOp<DateTimeValue> {
         value.from_olap_datetime(*reinterpret_cast<const uint64_t*>(data));
         return bloom_filter.test(Slice((char*)&value, sizeof(DateTimeValue)));
     }
+    void insert(BloomFilterAdaptor& bloom_filter, const void* data) const {
```

Comment:
we should do the unify job should in function `find_olap_engine`

Reply:
who invokes this `insert`?

Reply:
shall we do the unify job of all datatypes in one place? There are other data types whose storage format are different from their engine format.